decidim/decidim
# Redesign: remove Sections sidebar from forms

### :tophat: What? Why?

While reviewing the redesign, we found some Section sidebars that are too empty and are taking too much space. This PR removes them from the different forms of the admin panel.
 
### Related issues

- Fixes #11650

### Testing

1. Sign in as admin
2. Go to the listed pages
4. See that there are not Section sidebars

#### Listed pages

##### Admin
- [ ] Categories new: http://localhost:3000/admin/participatory_processes/ample-agree/categories/new
- [ ] Categories edit: http://localhost:3000/admin/participatory_processes/ample-agree/categories/1/edit
- [ ] Components new: http://localhost:3000/admin/participatory_processes/ample-agree/components/new?type=pages
- [ ] Components edit: http://localhost:3000/admin/participatory_processes/ample-agree/components/6/edit
- [ ] Space private participant new: http://localhost:3000/admin/participatory_processes/ample-agree/participatory_space_private_users/new 

##### Assemblies
- [ ] Assembly new: http://localhost:3000/admin/assemblies/new 
- [ ] Assembly edit: http://localhost:3000/admin/assemblies/minister-academy/edit


##### Conferences

- [ ] Conference new: http://localhost:3000/admin/conferences/new
- [ ] Conference edit: http://localhost:3000/admin/conferences/brown-widen/edit 

##### Elections

- [ ] Trustee new: http://localhost:3000/admin/participatory_processes/ample-agree/components/10/manage/trustees/new 
- [ ] Voting new: http://localhost:3000/admin/votings/new
- [ ] Voting edit: http://localhost:3000/admin/votings/horizon-defend/edit

##### Initiatives

- [ ] Initiative edit: http://localhost:3000/admin/initiatives/i-1/edit
- [ ] Initiative type new: http://localhost:3000/admin/initiatives_types/new 
- [ ] Initiative type edit: http://localhost:3000/admin/initiatives_types/1/edit 

##### Processes

- [ ] Process new: http://localhost:3000/admin/participatory_processes/new
- [ ] Process edit: http://localhost:3000/admin/participatory_processes/ample-agree
- [ ] Process group new: http://localhost:3000/admin/participatory_process_groups/new
- [ ] Process group edit: http://localhost:3000/admin/participatory_process_groups/1/edit
- [ ] Steps new: http://localhost:3000/admin/participatory_process_groups/1/edit
- [ ] Steps edit: http://localhost:3000/admin/participatory_processes/ample-agree/steps/1/edit
- [ ] Process types new: http://localhost:3000/admin/participatory_process_types/new
- [ ] Process types edit: http://localhost:3000/admin/participatory_process_types/1/edit 
- [ ] Process user role new: http://localhost:3000/admin/participatory_processes/ample-agree/user_roles/new
- [ ] Process user role edit: http://localhost:3000/admin/participatory_processes/ample-agree/user_roles/1/edit

### :camera: Screenshots

#### Before
![Screenshot of the New process type page](https://github.com/decidim/decidim/assets/717367/431eee64-4b68-4335-8f56-02d8d557be79)

#### After

![Screenshot of the New process type page](https://github.com/decidim/decidim/assets/717367/6df3a5b7-3d75-4b8c-8c7a-12e5859d2714)

:hearts: Thank you!
